### PR TITLE
tdx/docker/client-image

### PR DESCRIPTION
**Pull request by Le-Zheng**
## Description

ppml/tdx/docker/client-image: 
 - Dockerfile: from bigdl-k8s 
 - CLI tool 
 - client yaml
 - driver/executor pod template
 - kata runtime deployment

### CODE REVIEW

File: ppml/tdx/docker/client-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM intelanalytics/bigdl-k8s
+
+ADD ./spark-submit-with-ppml-tdx-local.sh /opt/spark/work-dir/ppml-tdx/spark-submit-with-ppml-tdx-local.sh
+ADD ./spark-submit-with-ppml-tdx-k8s.sh /opt/spark/work-dir/ppml-tdx/spark-submit-with-ppml-tdx-k8s.sh
+ADD ./pod-template.yaml /opt/spark/work-dir/ppml-tdx/pod-template.yaml
+ADD ./tdx-client.yaml /opt/spark/work-dir/ppml-tdx/tdx-client.yaml
+
+ENTRYPOINT [ "/opt/entrypoint.sh" ]
```
Comment (glorysdj) [inline ppml/tdx/docker/client-image/Dockerfile:8]: why need entrypoint?

File: ppml/tdx/docker/client-image/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Build bigdl-tdx image
+```bash
+docker build \
+  --build-arg http_proxy=.. \
+  --build-arg https_proxy=.. \
+　--build-arg no_proxy=.. \
+  --rm --no-cache -t bigdl-tdx-client-spark-3.1.2:2.1.0-SNAPSHOT .
```
Comment (glorysdj) [inline ppml/tdx/docker/client-image/README.md:7]: add intelanalytics/XXX?

File: ppml/tdx/docker/client-image/spark-submit-with-ppml-tdx-k8s.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+if [ -z $RUNTIME_K8S_SPARK_IMAGE ]; then
+    echo "Please set BIGDL_HOME environment variable"
+    exit 1
+fi
+
+default_config="--conf spark.kubernetes.authenticate.driver.serviceAccountName=$RUNTIME_K8S_SERVICE_ACCOUNT \
+        --conf spark.kubernetes.container.image=$RUNTIME_K8S_SPARK_IMAGE \
+        --conf spark.kubernetes.executor.deleteOnTermination=false \
+        --conf spark.network.timeout=10000000 \
+        --conf spark.executor.heartbeatInterval=10000000 \
```
Comment (glorysdj) [inline ppml/tdx/docker/client-image/spark-submit-with-ppml-tdx-k8s.sh:23]: these timeout is too large, do we need for tdx?

Comment (Le-Zheng) [reply ppml/tdx/docker/client-image/spark-submit-with-ppml-tdx-k8s.sh]: Removed. We do not need on tdx.

File: ppml/tdx/docker/client-image/README.md (hunk 2 above)
Comment (Le-Zheng) [reply ppml/tdx/docker/client-image/README.md]: Done.

File: ppml/tdx/docker/client-image/Dockerfile (hunk 1 above)
Comment (Le-Zheng) [reply ppml/tdx/docker/client-image/Dockerfile]: Tested and removed.

**Pull request merged**